apps/web/src/pages/integrations/components/multi-provider/CreateProviderInstanceSidebar.tsx

```diff
@@ -0,0 +1,163 @@
+import { ActionIcon, Group, Radio, Text } from '@mantine/core';
+import { colors, NameInput, Button } from '../../../../design-system';
+import { ArrowLeft, Close } from '../../../../design-system/icons';
+import { Controller, useForm } from 'react-hook-form';
+import { inputStyles } from '../../../../design-system/config/inputs.styles';
+import { useFetchEnvironments } from '../../../../hooks/useFetchEnvironments';
+import { ChannelTypeEnum, ICredentialsDto, IProviderConfig } from '@novu/shared';
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { useSegment } from '../../../../components/providers/SegmentProvider';
+import { createIntegration } from '../../../../api/integration';
+import { IntegrationsStoreModalAnalytics } from '../../constants';
+import { errorMessage, successMessage } from '../../../../utils/notifications';
+import { QueryKeys } from '../../../../api/query.keys';
+import { ProviderImage, Footer, FormStyled } from './SelectProviderSidebar';
+import { CHANNEL_TYPE_TO_STRING } from '../../../../utils/channels';
+
+export function CreateProviderInstanceSidebar({
+  onClose,
+  provider,
+  goBack,
+}: {
+  onClose: () => void;
+  goBack: () => void;
+  provider: IProviderConfig;
+}) {
+  const { environments, isLoading: areEnvironmentsLoading } = useFetchEnvironments();
+  const queryClient = useQueryClient();
+  const segment = useSegment();
+
+  const { mutateAsync: createIntegrationApi, isLoading: isLoadingCreate } = useMutation<
+    { res: string },
+    { error: string; message: string; statusCode: number },
+    {
+      providerId: string;
+      channel: ChannelTypeEnum | null;
+      credentials: ICredentialsDto;
+      active: boolean;
+      check: boolean;
+    }
+  >(createIntegration);
+
+  const { handleSubmit, control } = useForm({
+    shouldUseNativeValidation: false,
+    defaultValues: {
+      name: provider.displayName,
+      environmentId: environments?.find((env) => env.name === 'Development')?._id || '',
+    },
+  });
+
+  const onCreateIntegrationInstance = async (data) => {
+    try {
+      await createIntegrationApi({
+        providerId: provider?.id,
+        channel: provider?.channel,
+        credentials: {},
+        active: false,
+        check: false,
+      });
```

Comment:
environment and name need to be added after we have it enabled in the post endpoint